### PR TITLE
This is the first task from testing: it test sumator!

### PR DESCRIPTION
**Pull request by qnislavnachev**
Testing Sumator!

### CODE REVIEW

File: test/test/SumatorTest.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+public class SumatorTest {
+    Sumator math = new Sumator();
+
+    @Test
+    public void testSumatorWithNullNumbers() throws Exception {
+        assertTrue(math.sum("0", "0") == 0);
+    }
+
+    @Test
+    public void testSumatorNotNullNumbers() throws Exception {
```
Comment (mgenov) [inline test/test/SumatorTest.java]: Just pointing out that test names should not start with "test" as of junit 4.

File: test/test/SumatorTest.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+import org.junit.Test;
+import task1.Sumator;
+
+
+public class SumatorTest {
+    Sumator math = new Sumator();
+
+    @Test
+    public void SumWithZeroParameters() throws Exception {
```
Comment (mgenov) [inline test/test/SumatorTest.java]: Simple rule: Method names in java should start with lower case character.

Comment (mgenov) [inline test/test/SumatorTest.java]: As I know your environment need to be configured to use checkstyle plugin with the following configuration: 
https://github.com/clouway-incubator/codestyle/blob/master/codestyle.xml

File: src/task1/Demo.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package task1;
+
+public class Demo {
+    public static void main(String[] args) throws Exception {
```
Comment (EmilGeorgiev) [inline src/task1/Demo.java]: i think that this class is unnecessary. We have tests for this purpose.

File: src/task1/Sumator.java [hunk 4]
```diff
@@ -0,0 +1,10 @@
+package task1;
+
+public class Sumator {
+    public double sum(String a, String b) throws Exception {
+        if(a == null || b == null){
```
Comment (EmilGeorgiev) [inline src/task1/Sumator.java]: if we remove ```b == null``` i think that all test will pass

File: test/test/SumatorTest.java [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+        assertTrue(math.sum("0", "0") == 0);
+    }
+
+    @Test
+    public void sumWithoutZeroParameters() throws Exception {
+        assertTrue(math.sum("1", "3") == 4);
+    }
+
+    @Test(expected = NumberFormatException.class)
+    public void expectedNumberFormatException() throws Exception {
```
Comment (EmilGeorgiev) [inline test/test/SumatorTest.java]: .

File: test/test/SumatorTest.java [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+        assertTrue(math.sum("c", "5") == 5);
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void expectedIllegalArgumentException() throws Exception {
```
Comment (EmilGeorgiev) [inline test/test/SumatorTest.java]: test name should tell what case we will test not what will be result of test

**Comment by EmilGeorgiev** [comment]
LGTM after fix comments

**Pull request closed without merging**